# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sirkull/jamin/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/scenes.c.orig       2004-05-09 18:38:40.000000000 +0200
+++ ./src/scenes.c     2004-08-10 09:31:45.750402978 +0200
@@ -60,7 +60,6 @@
                          *LED_yellow = NULL, *LED_red = NULL;
 
 
-void set_EQ_curve_values ();
 
 
 /*  Initialize all scene related structures and get the widget addresses.  */
@@ -210,7 +209,7 @@
 
                         s_crossfade_to_state (&scene_state[i], -1.0f);
 
-                        set_EQ_curve_values ();
+                        set_EQ_curve_values (0, 0.0);
 
                         s_history_add_state (scene_state[i]);
